core/src/main/java/com/linecorp/armeria/common/util/StartStopSupport.java

```diff
@@ -146,75 +149,77 @@ public final CompletableFuture<V> start(@Nullable T arg, @Nullable U rollbackArg
         return start0(arg, rollbackArg, failIfStarted);
     }
 
-    private synchronized UnmodifiableFuture<V> start0(@Nullable T arg,
+    private UnmodifiableFuture<V> start0(@Nullable T arg,
                                                       @Nullable U rollbackArg,
                                                       boolean failIfStarted) {
-        if (closeable.isClosing()) {
-            return exceptionallyCompletedFuture(new IllegalStateException("closed already"));
-        }
+        lock();
```

Comment:
nit: indentation

Reply:
i fixed it!
[68f92d3](https://github.com/line/armeria/pull/4551/commits/68f92d32bfc5acab90baa030001b06defd368e5e)